python3-jupyterlab/requirements.txt

```diff
@@ -0,0 +1,47 @@
+bleach==2.1.3
```

Comment:
are all of these named dependencies? If these included transitive dependencies it is better to exclude those and let the named dependencies handle version upgrade for their dependencies.